oaknational/Oak-Web-Application
# fix(section): make view all lessons button work

## Description

Music year: 2013

- List of changes

on the lesson page for the browse journey the "view all lessons" button takes you back to the the lesson listing page

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2427--oak-web-application.netlify.thenational.academy/pupils/beta/programmes/maths-secondary-year-11-higher-l/units/circle-theorems-1-1521/lessons
2. Click on a lesson 
3. You should be able to nav back to this page from lesson

### Job Story

**When** I am on a pupil lesson,
**I want to the back button to navigate back to the previous page**
**So that when its the user journey pupuls can go back to the lesson listing page**.

### Implementation details

- Use the unit and programme slug to determine the href
- note that for users who have arrived here from the classroom the link stays the same and take you back to the classoom

```jsx
https://www.thenational.academy/pupils/l/cooking-and-nutrition-preparing-fruit-and-vegetables-2007/lessons/introduction-exploring-delicious-fruits-and-vegetables-crvk4t/overview
```

- link back to beta journey if user has come from beta journey

![Untitled](https://prod-files-secure.s3.us-west-2.amazonaws.com/67f80e06-bded-4efc-be7f-f17923df79ed/40237d0f-89ba-4bf2-bf04-dbe40db87358/Untitled.png)

### Acceptance criteria

- [ ]  back button is enabled and navigates back to unit page
